.github/ISSUE_TEMPLATE/BUG.md

```diff
@@ -21,13 +21,18 @@ labels: 'type: bug'
 <!-- Tell us what happens instead of the expected behavior -->
 
 ## Possible Solution
-<!-- Not obligatory, but suggest a fix/reason for the bug, -->
-<!-- or ideas how to implement the addition or change -->
+<!-- Not obligatory, but suggest a fix/reason for the bug,
+     or ideas how to implement the addition or change -->
```

Comment:
[nit] to be consistent with other comments:

```
<!--
  Not obligatory, but suggest a fix/reason for the bug,
  or ideas how to implement the addition or change
-->
```

Reply:
done